src/core/namespace.c

```diff
@@ -97,6 +97,14 @@ static const TargetMount protect_kernel_tunables_table[] = {
         { "/sys/fs/cgroup",             READWRITE,      false }, /* READONLY is set by ProtectControlGroups= option */
 };
 
+/* ProtectKernelModules= option */
+static const TargetMount protect_kernel_modules_table[] = {
+        { "/lib/modules",               INACCESSIBLE,   true  },
+        { "/lib64/modules",             INACCESSIBLE,   true  },
+        { "/usr/lib/modules",           INACCESSIBLE,   true  },
+        { "/usr/lib64/modules",         INACCESSIBLE,   true  },
```

Comment:
Hmm, so, the kernel modules are never in /usr/lib64. I am pretty sure we should only look for /usr/lib here, not the 64bit counterpart, after all the lib64 thing is only necessary for multilib implementations, and lib64 should pretty much only include .so libraries, and nothing else, in particular not kernels or their kmods.

I also think we should not add /lib/modules to the block list here, except if HAVE_SPLIT_USR is undefined. i.e. on joined /usr, we shouldn't bother with the dir at all...